website/server/models/group.js

```diff
@@ -376,6 +376,7 @@ schema.methods.removeGroupInvitations = async function removeGroupInvitations ()
   let userUpdates = usersToRemoveInvitationsFrom.map(user => {
     if (group.type === 'party') {
       user.invitations.party = {};
+      user.invitations.parties = [];
```

Comment:
this shouldn't remove only one invitation (matching the group variable) instead of all?